sdv/sequential/par.py

```diff
@@ -264,7 +264,9 @@ def auto_assign_transformers(self, data):
         # Ensure that sequence index does not get auto assigned with enforce_min_max_values
         if self._sequence_index:
             sequence_index_transformer = self.get_transformers()[self._sequence_index]
-            if sequence_index_transformer.enforce_min_max_values:
+            if sequence_index_transformer and getattr(
+                sequence_index_transformer, 'enforce_min_max_values', False
+            ):
```

Comment:
Actually can we also add a quick unit test that has a sequence index with a transformer but not the attribute?

Reply:
Added